pipeline_dp/report_generator.py

```diff
@@ -25,7 +25,7 @@ def report(self) -> str:
         if self._params is None:
             return ""
         title = f"Computing metrics: {[m.value[0] for m in self._params.metrics]}"
```

Comment:
Please m.value[0] -> m.value (and as a result will be full name of the metrics, not just first letter)

Reply:
Got it, looks much nicer now, thank you 🎉